crates/anvil/src/eth/backend/mem/mod.rs

```diff
@@ -178,14 +180,15 @@ impl Backend {
         env: Arc<RwLock<Env>>,
         genesis: GenesisConfig,
         fees: FeeManager,
-        fork: Option<ClientFork>,
+        fork: Arc<RwLock<Option<ClientFork>>>,
```

Comment:
this is redundant because the ClientFork is already an Arc type, see fields

Reply:
Right but it's wrapped in an `Option` which could be `None` and I need to make it `Some` which requires a mutable reference. So, then the whole `backend` would need to be mutable which it's not in `spawn` in `lib.rs`.